### ISSUE TITLE
dailymotion.com: detection

### ISSUE THREAD

**Issue by Harvey699** [uBlockOrigin/uAssets#26137]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.dailymotion.com/video/x928k7s`

### Category

detection

### Description

I can't watch DailyMotion.com b/c they detect ad blocker(s).

### Other extensions used

<details>

![image](https://github.com/user-attachments/assets/595b0727-6feb-44ee-9dc4-72d292d07b43)
![image](https://github.com/user-attachments/assets/8efa27c5-a64a-4c51-9b3e-e8a425b863f0)
![image](https://github.com/user-attachments/assets/6acc632e-4c09-467e-aac4-11a183461c7c)

</details>

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Chromium: 128
filterset (summary):
 network: 137388
 cosmetic: 49780
 scriptlet: 23103
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 40640-133, now
  ublock-badware: 11975-1, now
  ublock-privacy: 1512-23, now
  ublock-unbreak: 2507-1, now
  ublock-quick-fixes: 228-12, now
  easylist: 79080-603, now
  easyprivacy: 53272-755, now
  urlhaus-1: 19190-0, now
  plowe-0: 3538-0, now
filterset (user): [empty]
switchRuleset:
 added: [array of 2 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1747 ms (selfie)
 maxAssetCacheWait: 552 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 27
 no-large-media: true
 no-popups: true
 network:
  dm-event.net: 18
  hubvisor.io: 1
  imasdk.googleapis.com: 2
  privacy-mgmt.com: 1
  spot.im: 5
 extended:
  ##div[class^="DisplayAd"]
  ##div[class^="NewWatchingDiscovery__adSection"]
```
</details>

**Comment by Harvey699** [comment]
This only happens on DailyMotion.com from time to time.  It's been happening for the past 4 years I'd guess.

**Comment by stephenhawk8054** [comment]
Test again in a new browser profile with only uBO and its default settings.

**Comment by Harvey699** [comment]
OK. I'll do that.

**Comment by Harvey699** [comment]
Hi, It's working now but is it necessary to keep all of my other extensions off if I want to watch a show on DailyMotion?

**Comment by stephenhawk8054** [comment]
You need to find out yourself which extensions cause the issue on your side.

You didn't respect the prerequisites, and just tick on the box just for the sake of submitting the issue

> - [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)



> - [X] I am not using uBlock Origin along with other content blockers.

> - [X] I have verified that other extensions are not causing the issue.

> - [X] I did not answer truthfully to **ALL** the above checkboxes.

**Comment by Harvey699** [comment]
Well, I did the best that I could. I'm not a techie so that is why I came here for assistance. I didn't think I had to play whack a mole with my extensions so see if "maybe" I could sort this out. The way those questions are worded I couldn't answer truthfully since all of the questions had an asterisk  and I told the truth in the last question. I use many extensions so I was using UBlock Origin with the extensions I sent.I went on the page to find the duplicate issue but didn't see any inquiries about my situation or anything else for that matter.
